finished homework 6

theGrandeLatte opened the pull request:
thanks!!

theGrandeLatte committed:
finished homework 6

jamdon commented on routes/truckRoutes.js:
```diff
@@ -1,2 +1,83 @@
+var router = express.Router();
+
+//the food trucks
+var foodTruck = require('../models/truckModel');
```
This is technically a constructor function and should be capitalized.

jamdon commented on routes/truckRoutes.js:
```diff
@@ -1,2 +1,83 @@
+      } else {
+
+            //yay saved
+        response.status.send(newFoodTruck);
```
You're trying to chain off `status`, which is a function, when what you want to do is chain off it's return value, meaning you need to call it.

jamdon commented:
There's a problem in your POST route causing your server to crash. I've left an inline comment.

theGrandeLatte committed:
addressed crashing issue & route

updated crashing and change the route to point to _id since it appears
the html is now giving me that. (used to be name)

jamdon commented on routes/truckRoutes.js:
```diff
@@ -1,2 +1,83 @@
+
+.delete(function (request, response) {
+
+  foodTruck.findOneAndRemove({ name: request.params.name }, function(err){
```
This doesn't work. The parameter being sent is an id, not a name, which means that after you "delete" it, it will show up again when you refresh the page because it won't find a name that matches that id. The method you want to use is find ByIdAndRemove.

jamdon commented:
There's still an issue with your delete route. I've left a comment inline.

jamdon commented:
Grading as-is. 👍
